backend/data/src/main/kotlin/io/tolgee/configuration/tolgee/TolgeeProperties.kt

```diff
@@ -76,6 +76,8 @@ open class TolgeeProperties(
   @DocProperty(hidden = true)
   var ga4Tag: String? = null,
   @DocProperty(hidden = true)
+  var userSourceField: Boolean = false,
+  @DocProperty(hidden = true)
```

Comment:
`userSouce` is not very self-explanatory name, can you add description to the `DocProperty` annotation?